config/config/sc-config.yaml

```diff
@@ -424,17 +424,17 @@ thanos:
         cpu: 50m
         memory: 50Mi
       limits:
-        cpu: 500m
-        memory: 200Mi
+        cpu: 2000m
```

Comment:
It looks a bit strange that we have a request of 50m cpu but the limit at 2000m cpu. That is a huge difference. In your investigation doc it looks like you had a peak of 1000m cpu, why are you setting it to 2000m cpu here?

Reply:
The peak value written there is what levels it was when it crashed. To save the pod from crashing, I'm thinking that a doubling should do the job.